setup.cfg

```
@@ -53,7 +53,7 @@ install_requires =
     python-Levenshtein-wheels==0.13.1
     pytz==2020.1
     PyYAML==5.3.1
-    Red-Lavalink==0.6.0
+    Red-Lavalink==0.7.0
```

```suggestion
    Red-Lavalink==0.7.1
```
Bumping this to 0.7.1, ahead of the release of https://github.com/Cog-Creators/Red-Lavalink/pull/84